examples/correlation_matrices/auxiliaries.h

```diff
@@ -0,0 +1,73 @@
+
+#include <fstream>
+
+template <typename Point>
+void write_to_file(std::string filename, std::vector<Point> const& randPoints) {
+    std::ofstream out(filename);
+    auto coutbuf = std::cout.rdbuf(out.rdbuf()); //save and redirect
+    for(int i=0; i<randPoints.size(); ++i)
+        randPoints[i].print();
+    std::cout.rdbuf(coutbuf); //reset to standard output again
+}
+
+template<typename MT>
+MT rand_matrix(int n){
```

Comment:
What is the objective of this function? A random correlation matrix or just a random matrix with entries from [-1,1] for the non-diagonal elements?

Reply:
This function is removed.